ligand/urls.py

```diff
@@ -1,24 +1,28 @@
 from django.conf.urls import url
 from django.views.decorators.cache import cache_page
 from ligand import views
+from django.views.generic import TemplateView
+
 urlpatterns = [
     url(r'^$', cache_page(3600*24*7)(views.LigandBrowser.as_view()), name='ligand_browser'),
     url(r'^target/all/(?P<slug>[-\w]+)/$', views.TargetDetails, name='ligand_target_detail'),
     url(r'^target/compact/(?P<slug>[-\w]+)/$', views.TargetDetailsCompact, name='ligand_target_detail_compact'),
     url(r'^targets$', views.TargetDetails, name='ligand_target_detail'),
     url(r'^targets_compact', views.TargetDetailsCompact, name='ligand_target_detail_compact'),
     url(r'^targets_purchasable', views.TargetPurchasabilityDetails, name='ligand_target_detail_purchasable'),
-    url(r'^(?P<ligand_id>[-\w]+)/$', views.LigandDetails, name='ligand_detail'),
+    url(r'^(?P<ligand_id>[-\w]+)/details$', views.LigandDetails, name='ligand_detail'),
     url(r'^statistics', cache_page(3600*24*7)(views.LigandStatistics.as_view()), name='ligand_statistics'),
     url(r'^bias_statistics', cache_page(3600*24*7)(views.LigandBiasStatistics.as_view()), name='ligand_statistics'),
-    url(r'^experiment/(?P<pk>[-\w]+)/detail$', views.ExperimentEntryView.as_view()),
+
+    url(r'^biasedbrowser$', TemplateView.as_view(template_name='bias_browser.html')),
+    url(r'^biasedsubtypesbrowser$', TemplateView.as_view(template_name='bias_g_browser.html')),
+    url(r'^bias/api/$', views.BiasAPI.as_view(), name='api-rest'),
+    url(r'^biasg/api/$', views.GBiasAPI.as_view(), name='g-api-rest'),
+    url(r'^biasedbrowser/experiment/(?P<pk>[-\w]+)/detail$', views.ExperimentEntryView.as_view()),
+    url(r'^biasedsubtypesbrowser/experiment/(?P<pk>[-\w]+)/detail$', views.ExperimentEntryView.as_view()),
```

Comment:
Are:

biasedbrowser/experiment
biasedsubtypesbrowser/experiment

Already functional?
Perhaps comment them out for the time being as they are not yet done.

Reply:
they are already functional
As soon as new views go to server the links will be avail